source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -308,3 +464,23 @@ There are two ways to execute the ``bulkWrite()`` method:
 - Ordered, which performs the bulk operations in order until an error occurs, if any
 - Unordered, which performs all the bulk operations in any order and reports errors
   at the end, if any
+
+``MongoClient.bulkWrite()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When connecting to a deployment running {+mdb-server+} version 8.0 or later, you 
+can use the ``MongoClient.bulkWrite()`` method to perform bulk operations on multiple 
+databases and collections at once.
+
+This method uses the ``ClientNamespacedWriteModel`` and its methods
+``insertOne()``, ``updateOne()``, ``updateMany()``, ``replaceOne()``, 
+``deleteOne()``, and ``deleteMany()``.
```

Comment:
S: you can list this list of supported methods in the introduction to client bulk write. maybe around line 325 of the PR instead of only listing 2 methods